Phonebook.Frontend/src/app/shared/states/App.state.ts

```diff
@@ -93,4 +111,20 @@ export class AppState {
       sendFeedback: action.sendFeedback
     });
   }
+
+  @Action(SetTheme)
+  public setTheme(ctx: StateContext<AppStateModel>, action: SetTheme) {
+    const state = ctx.getState();
+    this.themeService.setTheme(action.theme)
+    ctx.setState({
+      ...state,
+      activeTheme: action.theme
+    });
+  }
+
+  @Action(InitTheme)
+  public initTheme(ctx: StateContext<AppStateModel>) {
+    const state = ctx.getState();
```

Comment:
I hope this ctx is stored in the cookie or something like this. So I can have my theme on the next day too :)

Reply:
yep it is